litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-LIMIT-RESOURCE/test.yml

```diff
@@ -155,18 +155,43 @@
           set_fact:
             node4_id: "{{ node_id[3].split('n')[1] }}"
 
-        - name: Labeling the nodes of the cluster
-          uri:
-            url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebsid }}/?action=selectopenebsnodes"
-            method: POST
-            url_username: "{{ username.stdout }}"
-            url_password: "{{ password.stdout }}"
-            force_basic_auth: yes
-            return_content: yes
-            body_format: json
-            body: '{"controlPlaneNodes":["{{node1_id}}"], "dataPlaneNodes":["{{node1_id}}","{{node2_id}}","{{node3_id}}","{{node4_id}}"]}'
-            status_code: 200
-          register: labelnodes
+        - block:
+          
+          - name: Get the node4_id
+            set_fact:
+              node5_id: "{{ node_id[4].split('n')[1] }}"
+          
+          - name: Labeling the nodes of the cluster
+            uri:
+              url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses/{{ openebsid }}/?action=selectopenebsnodes"
+              method: POST
+              url_username: "{{ username.stdout }}"
+              url_password: "{{ password.stdout }}"
+              force_basic_auth: yes
+              return_content: yes
+              body_format: json
+              body: '{"controlPlaneNodes":["{{node1_id}}",,"{{node2_id}}","{{node3_id}}","{{node4_id}}",{{node5_id}}], "dataPlaneNodes":["{{node1_id}}","{{node2_id}}","{{node3_id}}","{{node4_id}}","{{node5_id}}"]}'
```

Comment:
Are we labeling all of them as `control-plane` as well as `data-plane` nodes?

Reply:
Yes, we need to label all the nodes as `data-plane: true` so that we don't face any issue in openebs functional tests. For openebs functional test cases, we have requirement to create ndm pod all the nodes and it is possible only when we will label all the nodes as `data-plane: true`.  

There is no specific reason to label all the nodes as `control-plane:true`. But to be on the safer side I am labeling all the nodes as `control-plane:true`
@harshshekhar15